src/frameworks/drupal8/solr.md

```diff
@@ -1,7 +1,30 @@
 # Using Solr with Drupal 8.x
 
-## Requirements
+## Steps
+  - Enable Solr Service
+  - Enable the Solr relationship
+  - Download and enable the modules
+  - Configure Search API Solr
+
+### Enable Solr Service
+Different Solr versions are supported. Advanced configuration may be found at the [Solr Service documentation page](../../configuration/services/solr.md).
+
+Basically, uncomment or add these lines to your ``.platform/services.yaml``:
+```yaml
+solrsearch:
+    type: "solr:4.10"
```

Comment:
Docs should always use the latest version unless there's a specific reason not to.  Let's go full Solr 6.3.  Which of course means we will need to provide the "full" configuration since the default isn't working correctly... :-/